### DISCUSSION TITLE
Why q-table use native scroll?

### DISCUSSION THREAD

**Discussion by deepreshion** [quasarframework/quasar#16785] [General - Components / Directives / etc]
in my opinion, if q-scroll-area were used as a scroll, it would be much more convenient, maybe I didn’t see something in the documentation and it can be used instead of the native?

**Comment by yusufkandemir** [comment]
https://quasar.dev/vue-components/table#virtual-scrolling

**Reply by deepreshion** [reply]
![scroll](https://github.com/quasarframework/quasar/assets/72668881/7d664444-fb87-48c5-bb47-777c69d364fb)
this is a native scroll, I made a mistake in the description, I need a scroll like in the q-scroll-area component
